packages/server/package.json

```diff
@@ -47,17 +48,19 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.0.0",
     "@testing-library/user-event": "^13.2.1",
+    "@types/aws-lambda": "^8.10.83",
     "@types/express": "^4.17.12",
     "@types/hash-sum": "^1.0.0",
     "@types/ws": "^7.4.4",
+    "aws-lambda": "^1.0.6",
```

Comment:
I don't think this package is needed. We're only using the types from `aws-lambda`. In fact, I've _never_ used anything but `@types/aws-lambda`, and it looks like it's totally out of sync with this (8.x vs 1.x here)

Reply:
This blows my mind, the fact you sort of only need to implement types and not the lib. It makes total sense and I have never thought about it.